youtube_dl/options.py

```diff
@@ -203,10 +203,12 @@ def _comma_separated_values_options_callback(option, opt_str, value, parser):
     network.add_option(
         '--proxy', dest='proxy',
         default=None, metavar='URL',
-        help='Use the specified HTTP/HTTPS/SOCKS proxy. To enable experimental '
-             'SOCKS proxy, specify a proper scheme. For example '
-             'socks5://127.0.0.1:1080/. Pass in an empty string (--proxy "") '
-             'for direct connection')
+        help='Use the specified HTTP/HTTPS/SOCKS proxy. To enable HTTP/HTTPS '
+             'or experimental SOCKS proxy, specify a proper scheme. For example '
+             'protocol://hostname:port/ or protocol://username:password@hostname:port/ '
```

Comment:
1. It's called scheme, not protocol.
2. It's not an example. Example must use real arguments not parameters.